raiden/network/transport/matrix/client.py

```diff
@@ -494,7 +499,11 @@ def _handle_response(self, response: Dict[str, Any], first_sync: bool = False) -
         # Handle presence after rooms
         for presence_update in response["presence"]["events"]:
             for callback in list(self.presence_listeners.values()):
-                self.call(callback, presence_update)
+                self._worker_pool.spawn(
+                    self.call, callback, presence_update, next(self._presence_update_ids)
+                )
+        # Collect finished greenlets and errors without blocking
+        self._worker_pool.join(timeout=0, raise_error=True)
```

Comment:
hummm, ideally this should be done in the `get`, but `GMatrixClient` is not a `Runnable`